dashboard/app/models/census/census_summary.rb

```diff
@@ -28,4 +28,161 @@ class Census::CensusSummary < ApplicationRecord
   enum teaches_cs: TEACHES
 
   validates_presence_of :audit_data
+
+  def self.submission_teaches_cs(submission, is_high_school)
```

Comment:
how about making this a question mark method?

Reply:
Yes, I'll make that change.